.gitlab-ci.yml

```diff
@@ -64,5 +64,7 @@ behave:
 
 example-django:
   extends: .cookiecutter
-  script:
-  - tests/field/example-django.sh
+  variables:
+    FIELDTEST_CHECKS: bandit,flake8,pylint,safety,kubernetes
+    FIELDTEST_TESTS: py37,behave
+  script: tox -e fieldtest -- django
```

Comment:
Hmmm, that looks okay. Actually, it makes me think about whether we should encapsulate the execution details within the fieldtest script or have them explicit, like here.

I feel, there must be a way to make this even simpler. Let's talk at the office... :speaking_head:

Reply:
OK, we have changed it back to calling the script directly, as we discussed.